README.md

```diff
@@ -8,14 +8,54 @@ In order to demonstrate the utility of QSC algorithms the [Open Quantum Safe (OQ
 
 ### Components
 
-The install image conatins following QSC-enabled components (all installed in `/opt/oqssa`):
+The build script compiles following QSC-enabled components (all installed in `$HOME/opt/oqssa`):
 
 - liboqs: All NIST-round 2 competition algorithms
 - openssl: QSC-enabled OpenSSL 1.1.1g library and utility applications
```

Comment:
we can change 1.1.1g to 1.1.1h 

since OQS-openssl has been merged with latest upstream 1.1.1h release